### ISSUE TITLE
PAN Coordinator Support

### ISSUE THREAD

**Issue by antmerlino** [RIOT-OS/RIOT#6632]
I've seen this mentioned in a few different issues, but it seems there is no stand-alone issue regarding support for PAN configuration via a coordinator. 

I am interested in working on this feature but am unsure if I'd be able to do it alone without at minimum guidance, or more likely, actual help writing it.  Are there reasons that this hasn't been completed yet that I may not be considering?  Is there a general interest for this?

**Comment by smlng** [comment]
Good point bringing this up (again), IIRC this is also an ongoing discussion for linux-wpan, maybe you chat with @alexaring on that topic, too.

**Comment by PeterKietzmann** [comment]
@antmerlino I would say the general interest exists and there is no explicit reason (besides of time resources) why it hasn't been implemented yet. Concerning guidance I would say just start and come around with concrete questions in your PR, here or on the devel mailing list.

**Comment by kaspar030** [comment]
@antmerlino I'd be really happy to see PAN coordinator support!

**Comment by miri64** [comment]
Assigning @jia200x for now as he does a lot with PHY and MAC stuff lately.

**Comment by jia200x** [comment]
In order to support this we actually need to implement a IEEE802.15.4 MAC. It will come right after #13943 :)

**Comment by maribu** [comment]
@jia200x: Ping?
